SQL/SQL.sublime-syntax

```diff
@@ -181,15 +196,18 @@ contexts:
       comment: We should probably handle nested bracket pairs!?! -- Allan
       scope: punctuation.definition.string.begin.sql
       push:
+        - meta_include_prototype: false
         - meta_scope: string.regexp.modr.sql
         - match: '\}'
           scope: punctuation.definition.string.end.sql
           pop: true
         - include: string_interpolation
   string_escape:
+    - meta_include_prototype: false
```

Comment:
`meta_include_protoype` is not crucial for contexts which are not pushed/set on stack directly, but only included via `- include`.

Reply:
yep, I was just being defensive here in case this context is somehow pushed onto the stack directly in future or from a different syntax. Unlikely, I know :)